uli_init/simulate.py

```diff
@@ -704,40 +700,41 @@ def build_molecule(molecule, length, para_weight):
     sequence : list
         List of the configuration sequence of the finished compound
     """
-    f = open("{}/{}.json".format(COMPOUND_DIR, molecule))
+    f = open(f"{COMPOUND_DIR}/{molecule}.json")
     mol_dict = json.load(f)
     f.close()
```

Comment:
This can be a little more concise by doing
```suggestion
    with open(f"{COMPOUND_DIR}/{molecule}.json") as f:
        mol_dict = json.load(f)
```
the file is closed after the indented block

Reply:
True. I guess I generally prefer avoiding large sections of extra indented code if possible, especially in this case if it's just shaving off one line of code.  I wonder what Mr. PEP8 says :)

Reply:
True! Both do the exact same thing, so it's really a matter of preference.